rebar.config

```diff
@@ -304,7 +304,7 @@
 {dialyzer, [
             {warnings, [unknown]},
             {plt_apps, all_deps},
-            {base_plt_apps, [erts, kernel, stdlib, crypto, mnesia, trace_runner, rocksdb, mnesia_rocksdb]}
+            {base_plt_apps, [erts, kernel, stdlib, crypto, mnesia, trace_runner, rocksdb, mnesia_rocksdb, aesophia_aci_encoder]}
```

Comment:
OK, we can do this for now, but let's actually fix the typing in aesophia_aci_encoder later. Ok?

Reply:
We'll fix it definitely, but just out of this scope.